src/libs/colorpicker.c

```diff
@@ -160,6 +161,18 @@ static void _update_sample_label(dt_colorpicker_sample_t *sample)
       // Lab
       snprintf(text, sizeof(text), "%6.02f %6.02f %6.02f", CLAMP(lab[0], .0f, 100.0f), lab[1], lab[2]);
       break;
+
+    case 2:
+      // LCh
+      dt_Lab_2_LCH(lab, alt);
+      snprintf(text, sizeof(text), "%.02f %.02f %.02f", CLAMP(alt[0], .0f, 100.0f), alt[1], alt[2] * 360);
```

Comment:
all format should be : %2.02f as LCh values can be > 10. otherwise the number does not align well.